pyoos/parsers/ioos/one/describe_sensor.py

```diff
@@ -49,7 +56,7 @@ def get_ioos_def(self, ident, elem_type, ont):
 
     def __init__(self, element):
         """ Common things between all describe sensor requests """
-        if isinstance(element, str):
+        if isinstance(element, bytes):
```

Comment:
I am using `isinstance(element, (str, bytes))` and the tests are passing on both Python 2 and 3 :grimacing: 

However, since all we are doing here is testing if it is not already an element we could just:

`isinstance(element, etree._Element)`